lusidtools/cocoon/utilities.py

```diff
@@ -1327,3 +1327,38 @@ def update_value(d: typing.Union[dict, str], val: typing.Union[str, float]):
     d = val
 
     return d
+
+
+def enrich_existing_column(data_frame, mapping, field_name, enriched_column, constant_prefix):
+    # A column for name already exists and needs to be enriched
```

Comment:
This comment needs to be extended to include any provided field_name, previously it was only for instrument name explicitly.

Reply:
# A column for provided field_name already exists and needs to be enriched